modules/composer-ffmpeg/src/main/java/org/opencastproject/composer/impl/endpoint/ComposerRestService.java

```diff
@@ -629,6 +631,46 @@ public Response convertImage(@FormParam("sourceImage") String sourceImageXml, @F
     }
   }
 
+  /**
+   * #DCE OPC-29 Demuxes a track into multiple outputs
```

Comment:
Ideally we don't include private ticket numbers in upstream source contributions.  Likewise, this method is missing its description.